src/RemoteTech/RTLog.cs

```diff
@@ -26,17 +26,17 @@ public static class RTLog
 
         static RTLog()
```

Comment:
all of the changes here were just removing unnecessary qualifiers and fixing casing